CONTRIBUTING.md

```diff
@@ -1,23 +1,36 @@
 # Contributing
 
-Everyone is very welcome to contribute on the codebase of Remix. Please join our [Discord](https://discord.gg/mh9hFCKkEq) in case of any queries.
+Everyone is welcome to contribute Remix's codebase. Please join our [Discord](https://discord.gg/mh9hFCKkEq) in case of any queries.
```

Comment:
I think the sentence should read "Everyone is welcome to contribute to Remix's codebase"